papers/doc/7-systems/n163.md

```diff
@@ -1,8 +1,8 @@
 # Namco 163 (also called N163, Namco C163, Namco 106 (sic), Namco 160 or Namco 129)
 
-This is one of Namco's NES mappers, with up to 8 wavetable channels. It has also 128 byte of internal RAM, and both channel register and wavetables are stored here. Wavetables are variable size and freely allocable anywhere in RAM, it means it can use part of or continuously pre-loaded waveform and its sequences in RAM. But waveform RAM area becomes smaller as more channels are activated; as channel registers consumes 8 bytes for each channel. You must avoid conflict with channel register area and waveform for avoid broken channel playback.
+This is one of Namco's NES mappers, with up to 8 wavetable channels. It has also 256 nibbles (128 bytes) of internal RAM, and both channel registers and wavetables are stored here. Wavetables are variable in size and freely allocable anywhere in RAM, it means it can use part of or continuously pre-loaded waveform and its sequences in RAM. At least 64 bytes can dedicated to waves, with more available if not all channels are used - waveform RAM area becomes smaller as more channels are activated, since channel registers consume 8 bytes for each channel. You must avoid conflict with channel register area and waveform to avoid broken channel playback.
 
-It outputs only a single channel at clock; so its sound quality gets more crunchy as more channels are activated.
+Namco 163 does not internally mix its channels. Instead, each channel is output one at a time, when multiple channels are used it will cycle between them; so its sound quality gets more crunchy as more channels are activated.
```

Comment:
1. You copied this verbatim from NESdev Wiki
2. A comma which separates two unrelated contexts or starts a new sentence must be a dot or a semicolon.

Reply:
Reworded it a bit, let me know if it's good now (can't help that nesdev wiki explained it in the best way w)